undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java

```diff
@@ -342,4 +342,8 @@
     @LogMessage(level = WARN)
     @Message(id = 80, value = "Valves are no longer supported, %s is not activated.")
     void unsupportedValveFeature(String valve);
+
+    @LogMessage(level = INFO)
+    @Message(id = 81, value = "Distributable is disabled in web-fragment.xml of module %s, while it is enabled in web.xml")
```

Comment:
I think it would be good if this message explicitly stated that the resulting deployment will not be distributable.